src/components/NavBar.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import {makeStyles, Theme} from '@material-ui/core/styles';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import Button from '@material-ui/core/Button';
+import IconButton from '@material-ui/core/IconButton';
+import MenuIcon from '@material-ui/icons/Menu';
+import {useHistory} from "react-router-dom";
+
+const useStyles = makeStyles((theme: Theme) => ({
+    root: {
+        flexGrow: 1,
+    },
+    menuButton: {
+        marginRight: theme.spacing(2),
+    },
+    title: {
+        flexGrow: 1,
+    },
+}))
+
+export default function NavBar() {
+    const classes = useStyles()
+    let history = useHistory()
+
+    let logButton;
+
+    if (localStorage.getItem("jwt")) {
+        logButton = <Button color="inherit" onClick={() => {
+            history.push("/login")
+        }}>Login</Button>
+    } else {
+        logButton = <Button color="inherit" onClick={() => {
+            localStorage.clear()
+        }}>Logout</Button>
+    }
```

Comment:
Good direction but creating a variable is not necessary, use {} with "?" conditional statement instead in your destined place inside component return function.
Example { localStorage.getItem("jwt") ? <Button1/> : <Button2/>}

Reply:
co pan mówisz ja nie rozumiem

Reply:
gosciu to nie po polsku co ty mowisz nie rozumiem

Reply:
omg polack in my lab team

Reply:
gg surr

Reply:
gg surr 15